packages/grafana-ui/src/components/DataLinks/FieldLinkList.tsx

```diff
@@ -28,25 +33,48 @@ export function FieldLinkList({ links }: Props) {
   return (
     <>
       {internalLinks.map((link, i) => {
-        return <DataLinkButton key={i} link={link} />;
+        return (
+          <MaybeErrorBadge link={link} key={i}>
+            <DataLinkButton link={link} />
+          </MaybeErrorBadge>
+        );
       })}
       <div className={styles.wrapper}>
         <p className={styles.externalLinksHeading}>External links</p>
         {externalLinks.map((link, i) => (
-          <a key={i} href={link.href} target={link.target} className={styles.externalLink}>
-            <Icon name="external-link-alt" />
-            {link.title}
-          </a>
+          <MaybeErrorBadge link={link} key={i}>
+            <a href={link.href} target={link.target} className={styles.externalLink}>
+              <Icon name="external-link-alt" />
+              {link.title}
+            </a>
+          </MaybeErrorBadge>
         ))}
       </div>
     </>
   );
 }
 
+type MaybeErrorBadgeProps = {
+  link: LinkModel<Field>;
+};
+
+/**
+ * @internal
+ */
+const MaybeErrorBadge: React.FC<MaybeErrorBadgeProps> = (props) => {
+  const { link, children } = props;
+  return link.origin.config.links && link.origin.config.links[0].error ? (
+    <Badge text={link.title} tooltip={link.error} color="red" icon="external-link-alt" />
```

Comment:
Although we have a tooltip here, we cannot show it. Because the exemplar popup is in the tooltip container already. I couldn't find a way to show it.